README.md

```diff
@@ -0,0 +1,80 @@
+# STOCKIST WEB APP - REACT CAPSTONE PROJECT
+
+The project, a frontend web application, catalogue of financial values using [Stock API](https://financialmodelingprep.com/developer/docs/)  and collect the data from my React Application.
+
+The application features includes:
+- Listing of Most Active Stocks
+- Listing of Most Gainer's Stocks
+- Listing of Most Loser's Stocks
+- Filter between those 3, most active, gainers and losers
+- Open individual stock item
+- Search for a particular stock item, example 'AAPL'
+- Listing of the lastest stock news
+- Dynamic routes for individual stock item, example `/stocks/AAPL`
+
+![screenshot](./stockist.gif)
+
+## Live Demo
+[Live Demo](https://rebel216.github.io/Capstone-React-Finance/)
```

Comment:
- [ ] please kindly deploy with `heroku `or `netlify` as with github pages it keeps showing error 404.